GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,13 +8,64 @@
 
 import UIKit
 
-class ViewController: UIViewController {
 
+
+
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    @IBOutlet weak var tableviewController: UITableView!
+    
+    var episode = GOTEpisode.allEpisodes
+    
+    var gotArray = GOTEpisode.filteredSeasons()
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return gotArray[section].count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let oddCell = tableView.dequeueReusableCell(withIdentifier: "gotCell1") as! OddSeasonsTableViewCell
+        oddCell.oddTitleLabel.text = gotArray[indexPath.section][indexPath.row].name
```

Comment:
Better practices are, before your guard the cell, you can assign one variable to the single value of the data of your cell:
var dataToSetOnEachCell = gotArray[indexPath.section][indexPath.row]
e.g
cell.episodeLabel.text = dataToSetOnEachCell.name 
Then you can use this same value when you check on the identifier